mage/changelog.md

```diff
@@ -4,6 +4,21 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v1.5 - Dec 20, 2022
+
+### Major Features and Improvements
+
+- Added support for ancestors, descendants, topological sort, connecting nodes, and chaining nodes.
+  [#180](https://github.com/memgraph/mage/pull/180)
+- Aligned C++ API with Memgraph 2.5
+  [#184](https://github.com/memgraph/mage/pull/184)
+- Made Graph Coloring output vertices and integers instead of strings
+  [#177](https://github.com/memgraph/mage/pull/177)
+
+### Bug Fixes
+- Enable module reset to be able to train and evaluate without shutting down database, enable working with class labels which don't start from 0, and fix potential early stopping due to low limit in the Node classification module.
+  [#173](https://github.com/memgraph/mage/pull/173)
```

Comment:
By DOING THIS you can now train and evaluate WHAT without shutting down the database
Class labels can now start from 0
Low limit of WHAT no longer prematurely stops WHAT while running the Node classification module